server/api/db/sqldb/db.py

```diff
@@ -2416,13 +2416,20 @@ def delete_schedules(
             main_table_identifier_values=names,
         )
 
-    def align_schedule_labels(self, session: Session):
+    def align_schedule_labels(self, session: Session, enrich_kind: bool = False):
         schedules_update = []
         for db_schedule in self.list_schedules(session=session, as_records=True):
             schedule_record = self._transform_schedule_record_to_scheme(db_schedule)
             db_schedule_labels = {
                 label.name: label.value for label in db_schedule.labels
             }
+            if (
+                enrich_kind
+                and mlrun_constants.MLRunInternalLabels.kind not in db_schedule_labels
+            ):
+                # We will assume that if the kind of schedule is empty, it should be job
+                # because currently we only support job schedules
```

Comment:
I don't think the comment is accurate. It should say something like - the current schedule kind is miss-used as it is always job. We need to enrich it for the counters but it needs to be fixed. Please rephrase a bit and mark as TODO